hash/megacdj.xml

```diff
@@ -5157,6 +5157,92 @@ Black screen, requires sub CPU to be overclocked by 1.5x
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 19).bin" size="1117200" crc="421f466c" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 20).bin" size="13032432" crc="7fc3c5a7" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 21).bin" size="10499328" crc="4b6cc7ce" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 22).bin" size="9885456" crc="dad1304f" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 23).bin" size="11228448" crc="c73e18cb" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 24).bin" size="10861536" crc="57a73fc5" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 25).bin" size="13220592" crc="58acd55f" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 26).bin" size="11663568" crc="bd80964d" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 27).bin" size="11440128" crc="ac0f53d6" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 28).bin" size="11531856" crc="82dc6f50" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 29).bin" size="11195520" crc="4df9fad1" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 30).bin" size="11717664" crc="0cd8fd9b" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 31).bin" size="11865840" crc="18b4441d" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 32).bin" size="11376624" crc="356a93b0" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 33).bin" size="12517344" crc="1b43de31" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 34).bin" size="11983440" crc="7bd0c837" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 35).bin" size="16186464" crc="950b6a34" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 36).bin" size="14866992" crc="108f5a17" />
+		<rom name="Heavenly Symphony - Formula One World Championship 1993 (Japan) (Demo) (Track 37).bin" size="43267392" crc="e55cb55b" />
+		-->
+		<description>Heavenly Symphony - Formula One World Championship 1993 Hibaihin (Japan)</description>
+		<year>1994</year>
+		<publisher>Sega</publisher>
+		<info name="serial" value="SG-6034"/>
+		<info name="disk_ring" value="SG-6034-R1M V"/>
+		<info name="alt_title" value="ヘブンリー シンフォニー フォームラワン ワールド チャンピオンシップ 1993 非売品"/>
```

Comment:
Why does this software list use `disk_ring` info elements when megacd.xml uses `ring_code`?  It’s not even consistent between the two Mega CD lists.  Also, which code are these supposed to represent?  Mastering code?  Matrix?  Mastering SID?  Mould SID?  A specific info element should be used for each (e.g. cdi.xml uses `cd_matrix` for matrix codes).

Reply:
Oh, sorry, I didn't know there were several options for metadata. My `neocd,xml` submission will also need correcting. Is there a list somewhere that I can check, for reference, or did you just mention them all?

I'll fix all of these later today, sorry.

Reply:
No need to be sorry, these three lists were a mess before you got to them.

There’s a little info on the different “ring codes” here: https://www.rushmediaprint.com/cd-matrix-code-and-sid-codes/

I suspect all the codes in the three Mega CD software lists are matrix codes.  I’d suggest changing all three to use `cd_matrix` to match cdi.xml.

Reply:
Allright, done. I also removed an extra tag I left in by accident.